cmd/clusterctl/client/init.go

```diff
@@ -73,6 +73,10 @@ type InitOptions struct {
 	// IgnoreValidationErrors allows for skipping the validation of provider installs.
 	// NOTE this should only be used for development
 	IgnoreValidationErrors bool
+
+	// allowMissingProviderCRD allows listing of images if the provider CRD is missing in case of --list-images but make sure provider CRD is
+	// installed in the standard init process.
```

Comment:
```suggestion
	// allowMissingProviderCRD is used to allow for a missing provider CRD when listing images.
	// It is set to false to enforce that provider CRD is available when performing the standard init operation.
```

Reply:
@aniruddha2000 Do you have time to get to this?

Reply:
Sorry I was on a vacation. I pushed the changes.